pkgs/development/python-modules/mando/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+# Python deps
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "mando";
+  version = "0.7.1";
```

```suggestion
  version = "0.7.1";
  pyproject = true;
```
Could you please add this parameter?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters